src/common/mfu_io.h

```diff
@@ -121,21 +165,14 @@ int mfu_fsync(const char* file, int fd);
 /* get current working directory, abort if fail or buffer too small */
 void mfu_getcwd(char* buf, size_t size);
 
+int daos_mkdir(const char* dir, mode_t mode, mfu_file_t* mfu_file);
 /* create directory, retry a few times on EINTR or EIO */
 int mfu_mkdir(const char* dir, mode_t mode);
+int mfu_file_mkdir(const char* dir, mode_t mode, mfu_file_t* mfu_file);
 
 /* remove directory, retry a few times on EINTR or EIO */
 int mfu_rmdir(const char* dir);
 
-/* open directory, retry a few times on EINTR or EIO */
```

Comment:
For organization, let's move the opendir, closedir calls back down to this "Directories" section.